edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+
+
+## Edge device
+
+An edge device is a device which has the capability to enter into service provider core networks or connected to other smart devices to share the workload of a certain scenario. Examples include routers, routing switches, terminals and other devices. In those cases, ddge devices provide connections into carrier and internet service provider networks. An edge device doesn't need to be at connected state at all time, like mobile phones and smart cameras which can still work at offline state, but sharing of these data requires connectivity between those devices.
+
+In general, edge devices are normally routers and terminals that provide authenticated access to faster, more efficient backbone and core networks. Edge device like edge station can handle different kinds of works, it connect to cameras, sensors and other smart devices and is has access to the upper level core devices, often include Quality of Service (QoS) and multi-service functions to manage different types of traffic.
+
+## ONNX Edge profile
+
+The ONNX Edge profile is described with the following:
+
+1. Attributes / characteristics: Power, Size, accuracy, speed, Security, etc.
+
+2. ONNX operations subset
+
+3. Other ONNX related limitations 
+
+Work to define strict subsets of the operator sets which apply to edge devices which avoid complex operations imposed by limitations on edge devices with high computational complexity. Exclude use-cases not applicable to the edge (e.g. large scale training scenarios)
+
+
+### Edge profile attributes
+
+#### Accuracy
+
+The ONNX model zoo collected many wide-used models for image classification, face recognition and image segmentation. While the accuracy number varies, the top-5 accuracy seems acceptable with VGG, resent and mobile net are all above or close to 90%. Accuracy is critical for scenarios like smart city, face recognition and smart monitoring. If put these models in edge device in actual use, primary work is to keep the accuracy still in a decent level while compressing the model and changing ops to low-bit version. Detailed work can be achieved cooperating with the quantization WG.
+
+#### Size
+
+Size here refers to both the model on-disk storage size and the memory that the runtime reqiures to run the inference. In ONNX 1.4 release, support for large models (larger than 2GB) and store the data externally is added as a new feature. But in edge device, we do not want such huge models running and consuming the precious memory. There will always be compromises between performance, speed, cost and consumption. There are many ways to compress our models while keeping the accuracy like changing to quantized ops, model compression etc. 
```

Comment:
Proposing to delete "There will always be compromises between performance, speed, cost and consumption."  That is of larger scope than this paragraph. 
I proposed intro paragraph for  "Edge profile attributes" section, where we could expand on compromises that can be made. Things like model size / compression/ quantification trade-off vs. accuracy ...